parts/kubernetesmastercustomdata.yml

```diff
@@ -7,8 +7,30 @@ packages:
  - jq
  - traceroute
 
+disk_setup:
+  /dev/sdc:
+      table_type: gpt
+      layout: true
+      overwrite: true
+
+fs_setup:
+- special:
+  cmd: /sbin/mkfs.ext4 /dev/sdc1 -F -E lazy_itable_init=1,lazy_journal_init=1
```

Comment:
It's interesting that this is the fix... this is what we had to do in k8s too because mkfs.ext4 was being awfully slow in certain conditions.